content/matplotlib/concepts/pyplot/terms/subplots/subplots.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.subplots()'
+Description: It create a figure and a set of subplots. This function makes it easy to generate layouts of subplots, including the enclosing figure object.
+Subjects:
+  -  'Data Science'
+  -  'Data Visualization'
+  -  'Data Analytics'
+Tags:
+  -  'Graphs'
+  -  'Libraries'
+  -  'Matplotlib'
+CatalogContent:
+  -  'learn-python-3'
+  -  'visualize-data-with-python'
+  -  'machine-learning'
+---
+
+The **'pyplot.subplots()'** function returns:
+
+- fig: _Figure_
+
+- ax: _ax_ can be either a signe Axes object, or an array of Axes objects if more than one subplot was created. The dimensions of the resulting array can be controlled with the squeeze keyword.
+
+## Syntax
+```pseudo
+pyplot.subplots(nrows, ncols)
+```
+
+The following parameters can be used:
```

Comment:
This table must be completely reworked. It appears that much of this is the language used in the Matplotlib documentation. Docs is all original content, we do not permit any plagiarism. Contributors flagged twice for plagiarism will not be allowed to submit entries. All the language below must be rephrased in your own words. In Docs generally, we strive to keep the explanations and relevant information as concise as possible. Please review other examples in docs if you need further guidance.

Reply:
@caupolicandiaz Yes I finished all the changes you asked me to make.